mapper/pojo-standalone/src/main/java/org/hibernate/search/mapper/pojo/standalone/cfg/StandalonePojoMapperSettings.java

```diff
@@ -146,6 +157,7 @@ private Radicals() {
 				MAPPING_PREFIX + "multi_tenancy.tenant_identifier_converter";
 		public static final String INDEXING_PLAN_SYNCHRONIZATION_PREFIX = "indexing.plan.synchronization.";
 		public static final String INDEXING_PLAN_SYNCHRONIZATION_STRATEGY = INDEXING_PLAN_SYNCHRONIZATION_PREFIX + "strategy";
+		public static final String DEFAULT_MASS_INDEX_CLEAN_OPERATION = "default_mass_index_clean_operation";
```

Comment:
Shouldn't this be `indexing.mass.default_clean_operation`, for consistency with `indexing.listeners`, `indexing.plan`, etc.?

Reply:
mm ok makes sense 👍🏻